src/app/starredentries/starredentries.component.html

```diff
@@ -1,6 +1,6 @@
 <app-header>
   <mat-icon>star</mat-icon>
-  Starred Tools and Workflows
+  Starred Tools, Workflows, and Organizations
```

Comment:
I think `Starred` or `My Starred` works better, as the tools, workflows, and orgs are mentioned in the tabs already.